common/cultures/cwp_latin_american_cultures.txt

```diff
@@ -193,7 +193,7 @@
 		Rojas
 		Ruiz_2
 		Salas
-		Sarmaniego_del_Castillo
+		Samaniego_del_Castillo
```

Comment:
Something suggested this

Reply:
Translations used that name but not english loc